src/Compute/Compute.Test/ScenarioTests/GalleryTests.ps1

```diff
@@ -402,3 +402,169 @@ function Test-GalleryCrossTenant
         Clean-ResourceGroup $rgname
     }
 }
+
+<#
+.SYNOPSIS
+Testing gallery image version commands
+#>
+function Test-GalleryImageVersion
+{
+    # Setup
+    $rgname = Get-ComputeTestResourceName;
+    $galleryName = 'gallery' + $rgname;
+    $galleryImageName = 'galleryimage' + $rgname;
+    $galleryImageVersionName = 'imageversion' + $rgname;
+
+    try
+    {
+        # Common
+        $loc = "southcentralus";
```

Comment:
Location should not be hardcoded - this makes resording tests difficult in the future.

Reply:
@markcowl Sometimes, when tests for new APIs are recorded, the APIs are only available for some regions, so we specify a location in the tests.  I will re-record these tests later.

Reply:
@hyonholee Using the pattern in which you get the list of locatiosn and look for a preferred location in that list *always* works, and is a long-established pattern.  Please use this pattern in the future, as we will no longer accept PRs after code complete that have issues like this.